pkg/monitor/sqsevent/asg-lifecycle-event.go

```diff
@@ -95,6 +95,11 @@ func (m SQSMonitor) asgTerminationToInterruptionEvent(event *EventBridgeEvent, m
 	}
 
 	interruptionEvent.PostDrainTask = func(interruptionEvent monitor.InterruptionEvent, _ node.Node) error {
+		_, err = m.continueLifecycleAction(lifecycleDetail)
+		if err != nil {
+			return fmt.Errorf("continuing ASG launch lifecycle: %w", err)
```

Comment:
quick question: the error message seems confusing here, should it be terminate/ ASG lifecycle instead of launch in `asgTerminationToInterruptionEvent` method ? can you modify this to make it more clear as we have same error message in `createAsgInstanceLaunchEvent` method. I will approve this and will do NTH release either today or tomorrow once this PR is merged. Thanks :)

Reply:
Sorry, this should be fixed now.